doc: Update list of known issues of Thread. (#5248)
nrfconnect/sdk-nrf: doc/nrf/known_issues.rst

Thread
======

.. rst-class:: v1-6-1 v1-6-0

> KRKNWK-10633: When using Link Metrics Enhanced ACK-based Probing the Thread Information Element contains fixed data instead of the proper Link Metrics data assessed for the acknowledged frame.

Reviewer: @canisLupus1313 , this comment is lost from the File changes due to Outdated tag. Please include the suggested formatting edits.

> KRKNWK-10633: When using Link Metrics Enhanced ACK-based Probing the Thread Information Element contains fixed data instead of the proper Link Metrics data assessed for the acknowledged frame.

Author: Fxed.

.. rst-class:: v1-6-1 v1-6-0

> KRKNWK-10467: The current implementation does not guarantee that all retransmitted frames will be secured when using the radio driver transmission security capabilities.

Author: Fixed

  For this reason, OpenThread retransmissions are disabled by default when the :option:`CONFIG_NRF_802154_ENCRYPTION` Kconfig option is enabled.
  You can enable the retransmissions at your own risk.

.. rst-class:: v1-6-1 v1-6-0

> KRKNWK-11037: Potential Inifinite loop while using `Udp::GetEphemeralPort` in OpenThread.

Author: Fixed.

>   Usage of `Udp::GetEphemeralPort` should be avoided.

Author: fixed.

Thread
======

.. rst-class:: v1-6-1 v1-6-0

> KRKNWK-10633: Link Metrics Enhanced ACK-based Probing does not work properly.

Reviewer 2: ```suggestion
KRKNWK-10633: When using Link Metrics Enhanced ACK-based Probing the Thread Information Element contains fixed data instead of the proper Link Metrics data assessed for the acknowledged frame.
```

Author: Fixed

 **Workaround:** Not available, issue is fixed in newer version.

.. rst-class:: v1-6-1 v1-6-0

KRKNWK-10467: The current implementation does not guarantee that all retransmitted frames will be secured when using the radio driver transmission security capabilities.
  For this reason, OpenThread retransmissions are disabled by default when the :option:`CONFIG_NRF_802154_ENCRYPTION` Kconfig option is enabled.
  You can enable the retransmissions at your own risk.

> .. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0

Reviewer 2: ```suggestion
.. rst-class:: v1-6-1 v1-6-0
```
Older versions are not affected since the bug was introduced recently by: https://github.com/openthread/openthread/commit/257188b5e94687b63391602b532b08a4a9681a5d

Author: fixed.

Thread
======

.. rst-class:: v1-6-1 v1-6-0

KRKNWK-10633: Incorrect data when using ACK-based Probing with Link Metrics

> When using the ACK-based Probing enhanced with Link Metrics, the Thread Information Element contains fixed data instead of the correct Link Metrics data for the acknowledged frame.

Reviewer: Please indent the sentence.

Author: Fixed.

.. rst-class:: v1-6-1 v1-6-0

KRKNWK-10467: Security issues for retransmitted frames with Thread 1.2
  The Thread 1.2 current implementation does not guarantee that all retransmitted frames will be secured when using the transmission security capabilities of the radio driver.
  For this reason, OpenThread retransmissions are disabled by default when the :option:`CONFIG_NRF_802154_ENCRYPTION` Kconfig option is enabled.
  You can enable the retransmissions at your own risk.

.. rst-class:: v1-6-1 v1-6-0

> KRKNWK-11037:  Using ``Udp::GetEphemeralPort`` in OpenThread can potentially cause an infinite loop.

Reviewer: A sentence cannot be the header. Move the sentence to the next line (indented) and use something like:
```suggestion
KRKNWK-11037: ``Udp::GetEphemeralPort`` can case infinite loop
   Using ``Udp::GetEphemeralPort`` in OpenThread can potentially cause an infinite loop.
```
It's ok to be a bit repetitive here.

Author: Fixed.
